pkg/policy/api/ingress.go

```diff
@@ -108,12 +108,32 @@ type IngressRule struct {
 	//
 	// +optional
 	FromEntities EntitySlice `json:"fromEntities,omitempty"`
+
+	aggregatedSelectors EndpointSelectorSlice
+}
+
+// SetAggregatedSelectors creates a single slice containing all of the following
+// fields within the IngressRule, converted to EndpointSelector, to be stored
+// within the IngressRule for easy lookup while performing policy evaluation
+// for the rule:
+// * FromEntities
+// * FromCIDR
+// * FromCIDRSet
+func (i *IngressRule) SetAggregatedSelectors() {
```

Comment:
Add a comment indicating why we are not appending `FromEndpoints` here.

Reply:
Added.